op_crates/web/11_url.js

```diff
@@ -234,292 +174,51 @@
     }
 
     *keys() {
-      for (const [key] of this.#params) {
-        yield key;
+      for (let i = 0; i < this.#params.length; i++) {
```

Comment:
Why this change?

Reply:
I think this was a red herring for the iterator tests... reverted.